docs/usage/tracing.asciidoc

```diff
@@ -0,0 +1,72 @@
+[[ecs-tracing-usage]]
+==== Tracing Usage
+
+A trace describes the individual operations, and their causality, that ensue as a result of a single logical 
+operation. A distributed trace comprises operations across multiple distributed components, crossing process, 
+network, and security boundaries.
+
+For example, a distributed trace may be initiated when someone presses a button to start an action on a website - in this example, 
+the trace will represent calls made between the downstream services that handled the chain of requests initiated by this 
+button being pressed.
+
+A trace is made up of transactions and spans, where each transaction/span is one operation, activity, or part of the overall
+trace. A transaction is a special kind of span that is the highest level of work measured within a service. In Elastic APM, 
+transactions have additional attributes associated with them 
+(for more detail see https://www.elastic.co/guide/en/apm/get-started/current/transactions.html).
+
+Usually a transaction corresponds to the top level request to a component. Sometimes a component may break up processing into 
+multiple spans (and transactions) if needed. 
+
+The spans (and transactions) have a parent/child relationship with other spans and form a tree (or directed acylic grapph), 
```

Comment:
Is this an either-or? I think it'd be more clear if we said "also known as" instead.

```suggestion
The spans (and transactions) have a parent/child relationship with other spans and form a tree (also known as a directed acyclic graph), 
```